src/components/FavoriteList/FavoriteList.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { Title, Content, FavoriteList, FilterBar } from "./FavoriteList.styled";
+import favorites from '../../actions/favorites';
+import GifThumb from "../GifThumb";
+import SVG from "../SVG";
+
+export class FavoriteListComponent extends React.Component {
+	constructor(props) {
+		super(props);
+		this.input = null;
+		this.filter = this.filter.bind(this);
```

Comment:
+1 binding this in the constructor

another solution could be using arrow functions in the method